newdle/client/src/components/Home.js

```diff
@@ -2,21 +2,87 @@ import React from 'react';
 import {Button} from 'semantic-ui-react';
 import LoginRequired from './LoginRequired';
 import {useRouter} from '../util/router';
+import {Icon} from 'semantic-ui-react';
+
+import {ReactComponent as TeamIcon} from '../images/team.svg';
+import {ReactComponent as TimetableIcon} from '../images/timetable.svg';
+import {ReactComponent as DiscussIcon} from '../images/discuss-issue.svg';
+import {ReactComponent as GitHubIcon} from '../images/github.svg';
+
+import styles from './Home.module.scss';
 
 export default function Home() {
   const router = useRouter();
 
   return (
-    <>
-      <h2>Home</h2>
-      <LoginRequired
-        component={Button}
-        onClick={() => {
-          router.history.push('/new');
-        }}
-      >
-        Create newdle
-      </LoginRequired>
-    </>
+    <div>
+      <div className={styles.container}>
+        <h2>How does it work?</h2>
+        <ol className={styles.box}>
+          <li>
+            <TeamIcon className={styles.icon} />
+            Choose your participants
+          </li>
+          <li>
+            <TimetableIcon className={styles.icon} />
+            Set the time slots based on their availability
+          </li>
+          <li>
+            <DiscussIcon className={styles.icon} />
+            <strong>newdle</strong> will collect the answers!
```

Comment:
Does the `!` fit well here? The first two options don't have any punctuation at the end